daemon/cmd/kube_proxy_replacement.go

```diff
@@ -597,6 +621,52 @@ func disableNodePort() {
 	option.Config.EnableHostLegacyRouting = true
 }
 
+func markHostExtension() {
```

Comment:
Small comment why this function is needed for those who won't read the commit msg (also, when e.g., after refactoring `git blame` won't point to the original commit) is useful.